src/Makefile.am

```diff
@@ -167,6 +167,8 @@ libcvc4_la_SOURCES = \
 	proof/proof_output_channel.h \
 	proof/proof_utils.cpp \
 	proof/proof_utils.h \
+	proof/resolution_bitvector_proof.cpp \
+	proof/resolution_bitvector_proof.h \
```

Comment:
This file does not exist anymore after the switch to CMake. To resolve the merge conflict, it is fine to just remove those changes (i.e. nothing else needs to be done instead).